examples/conda-env-kernel-image/app-image-config-input.json

```diff
@@ -3,12 +3,12 @@
     "KernelGatewayImageConfig": {
         "KernelSpecs": [
             {
-                "Name": "conda-env-myenv-py",
+                "Name": "python3",
```

Comment:
super confused, how did it ever work before

Reply:
It must be `python3` so this is a good question...

Reply:
I got similar working correctly this week (`conda-env-mycoolenv-py`) - I believe this is the auto-detection mentioned in "Kernel discovery" section of the [custom image spec](https://docs.aws.amazon.com/sagemaker/latest/dg/studio-byoi-specs.html)? I logged docs feedback that they should clarify there what the "Name"s are for auto-detected conda environments.

I believe this is something that SageMaker or `jupyter_kernel_gateway` is doing? Because as [discussed here](https://stackoverflow.com/questions/39604271/conda-environments-not-showing-up-in-jupyter-notebook) seems like auto-setup of conda envs as Jupyter kernels is not automatic from conda itself?